src/content/docs/en/chromebook/installing.mdx

```diff
@@ -0,0 +1,78 @@
+  usually happens if the monitor has a resolution greater then 1080p. If your
+  Chromebox appeared to boot (lights on) but does not output anything, you can
+  attempt to continue without a display, or find a different one.
+</Alert>
+
+1.  Shut down the device, then insert a paper clip (a SIM ejector tool works great too!) into the recovery button (a small hole usually near the I/O, see your hardware documentation for exact location) and reboot. You will then be greeted with the "OS Verification is turned off" screen.
+2.  Plug in a keybaord and press <kbd>Ctrl</kbd>+<kbd>D</kbd>. You may be prompted with 'To turn OS verification off, press recovery button,' press recovery button.
+3.  A screen saying 'OS verification is off, press space to re-enable' should appear, press <kbd>Ctrl</kbd>+<kbd>D</kbd>.
+    Your system is now transitioning into Developer Mode! (This may take a few minutes).
+4.  After it has transitioned, press <kbd>Control</kbd>+<kbd>D</kbd> to boot back into ChromeOS as a developer. You are now ready for step 3.
+
+#### Developer Mode on ChromeOS Tablets:
+
+<Alert type="info">
+  This only supports x86 Chrome Tablets. Make sure your tablet is x86, or take a
+  peek at the [ARM Beta Program](armbeta).
+</Alert>
+
+1.  Shut down the device, then press and hold <kbd>Power</kbd>+<kbd>Volume-Up</kbd>+<kbd>Volume-Down</kbd> for 3-5 seconds, then release to invoke recovery mode. This is a bit finicky, and may take a few tries, but the screen will turn white and display a screen saying "Please insert a recovery USB stick." soon after you release the button.
+2.  To boot to Developer Mode, simultaneously press <kbd>Volume-Up</kbd>+<kbd>Volume-Down</kbd>, or navigate to 'Developer Options' with the volume buttons, and select it with the power button. Then select 'Boot From Internal Disk' using the volume buttons to navigate and power button to select.
+    Your system is now transitioning into Developer Mode! (This may take a few minutes).
+3.  After it has transitioned, again navigate to 'Developer Options,' then 'Boot From Internal Disk.' Plug in a keyboard, and you are now ready for step 3.
+
+##### If you have any problems with these steps, refer to [this guide](https://chromium.googlesource.com/chromiumos/docs/+/main/debug_buttons.md#Firmware-Keyboard-Interface) or request support in our [discord](https://discord.com/invite/5fdPuxTg5Q).
+
+### 3. Enable USB booting
+
+1. In ChromeOS Developer Mode, press <kbd>Ctrl</kbd>+<kbd>Alt</kbd>+<kbd>F2 [right arrow key]</kbd>.
+2. Type in 'Chronos' for the login (there is no password).
+3. Enter this command:
+
+```
+enable_dev_usb_boot
+```
+
+4. It should display `SUCCESS: Booting and self-assigned kernel from SSD/USB/SDCard slot in enabled` You are now ready for step 4.
+
+### 4. Boot from USB/SD
+
+1. Plug in your Ultramarine Chromebook installer and restart the device. If needed, press <kbd>Ctrl</kbd>+<kbd>alt</kbd>+<kbd>F1 [Left arrow key]</kbd> to leave the terminal.
```

Comment:
Alt is capped and just use a back arrow ←

Reply:
f2 isn't always forward arrow because newer cbs

Reply:
Then I'll say usually. 
Added in new commit.